test/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CalculatedGlarBreakdownSpec.scala

```diff
@@ -0,0 +1,189 @@
+        breakdown.propertyDisposalAmountLessCosts shouldBe AmountInPence(384_000_00)
+      }
+      "Return an accurate propertyAcquisitionAmountPlusCosts value" in {
+        breakdown.propertyAcquisitionAmountPlusCosts shouldBe AmountInPence(375_000_00)
+      }
+      "Return an accurate totalReliefs value" in {
+        breakdown.totalReliefs shouldBe AmountInPence(6_000_00)
+      }
+      "Return an accurate initialGainOrLoss value" in {
+        breakdown.initialGainOrLoss shouldBe AmountInPence(9_000_00)
+      }
+      "Return an accurate acquisitionOrRebased value" in {
+        breakdown.acquisitionOrRebased shouldBe AmountInPence(350_000_00)
+      }
+      "Return an accurate gainOrLossAfterReliefs value" in {
+        breakdown.gainOrLossAfterReliefs shouldBe AmountInPence(3_000_00)
+      }
+    }
+    "Given a rebase value while not using rebase" should {
+      val breakdown = CalculatedGlarBreakdown(
+        acquisitionPrice = AmountInPence(300_000_00),
+        acquisitionCosts = AmountInPence(15_000_00),
+        disposalPrice = AmountInPence(400_000_00),
+        disposalFees = AmountInPence(16_000_00),
+        privateResidentReliefs = AmountInPence(5_000_00),
+        lettingRelief = AmountInPence(1_000_00),
+        improvementCosts = AmountInPence(10_000_00),
+        shouldUseRebase = false,
+        rebasedAcquisitionPrice = Some(AmountInPence(350_000_00))
+      )
+      "Return an accurate propertyDisposalAmountLessCosts value" in {
+        breakdown.propertyDisposalAmountLessCosts shouldBe AmountInPence(384_000_00)
+      }
+      "Return an accurate propertyAcquisitionAmountPlusCosts value" in {
+        breakdown.propertyAcquisitionAmountPlusCosts shouldBe AmountInPence(325_000_00)
+      }
+      "Return an accurate totalReliefs value" in {
+        breakdown.totalReliefs shouldBe AmountInPence(6_000_00)
+      }
+      "Return an accurate initialGainOrLoss value" in {
+        breakdown.initialGainOrLoss shouldBe AmountInPence(59_000_00)
+      }
+      "Return an accurate acquisitionOrRebased value" in {
+        breakdown.acquisitionOrRebased shouldBe AmountInPence(300_000_00)
+      }
+      "Return an accurate gainOrLossAfterReliefs value" in {
+        breakdown.gainOrLossAfterReliefs shouldBe AmountInPence(53_000_00)
+      }
+    }
+    "shouldUseRebase is true but no rebasedAcquisitionPrice" should {
+      val breakdown = CalculatedGlarBreakdown(
+        acquisitionPrice = AmountInPence(300_000_00),
+        acquisitionCosts = AmountInPence(15_000_00),
+        disposalPrice = AmountInPence(400_000_00),
+        disposalFees = AmountInPence(16_000_00),
+        privateResidentReliefs = AmountInPence(5_000_00),
+        lettingRelief = AmountInPence(1_000_00),
+        improvementCosts = AmountInPence(10_000_00),
+        shouldUseRebase = true,
+        rebasedAcquisitionPrice = None
+      )
+      "Throw an accurate exception" in {
```

Comment:
⛏️ I wouldn't call it "accurate exception", just "throw `Exception`". I'd state somewhere that this is an edge case and wouldn't normally happen.